### PR TITLE
[ADD][sale_team_warehouse] New module added for set default warehouse from global default_get.

### PR DESCRIPTION
**Pull request by sabrinagnu**

### CODE REVIEW

File: sale_team_warehouse/models/sales_team.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from openerp.osv import fields, osv
+from openerp import api
+from openerp.models import BaseModel
+
+
+class InheritedCrmSaseSection(osv.Model):
+
+    """
+        ToDo
+    """
```
Comment (nhomar) [inline sale_team_warehouse/models/sales_team.py]: remove the todo if it will not be done.

File: sale_team_warehouse/models/sales_team.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    _inherit = "crm.case.section"
+    _columns = {
+        'default_warehouse': fields.many2one('stock.warehouse',
+                                             'Default Warehouse',
+                                             help='In this field can be '
+                                             'defined a default warehouse for '
+                                             'the related users of the team.'),
+    }
```
Comment (nhomar) [inline sale_team_warehouse/models/sales_team.py]: Use new api.

**Comment by dsabrinarg** [comment]
I will squash this ASAP.

**Comment by josemoralesp** [comment]
:+1: @dsabrinarg

**Comment by sabrinagnu** [comment]
@nhomar requested changes applied:
1.- index.html properly set
2.- Name of testing module initiating with "test_"
3.- read_group replaced by search

Green tests of travis.
Runbot ok (yellow for a warning in the main build reported and assigned here https://github.com/Vauxoo/yoytec/issues/214)

**Pull request merged**